src/app/settings/admin/rule-groups/component.ts

```diff
@@ -134,7 +134,7 @@ export class AdminSettingsRuleGroupsComponent implements OnInit, OnChanges, OnDe
       data: {
         title: 'Create Rule Group',
         mode: Mode.Add,
-        confirmLabel: 'Create',
+        confirmLabel: 'Add',
```

Comment:
Dialog's title is `Create` so I think it should also be changed to match the confirm button label.